SoftLayer/CLI/virt/os_available.py

```diff
@@ -0,0 +1,28 @@
+"""Get the all Os available."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@environment.pass_env
+def cli(env):
+    """Get the all Os available"""
+
+    table = formatting.KeyValueTable(['Id', 'KeyName', 'Description', 'Price', 'Setup Fee'])
```

Comment:
Add Monthly cost. The portal knows if you are building an hourly or monthly server, so it can just show you the relevant price. For the CLI we should show both, and Setup since the portal shows that too.
```suggestion
    table = formatting.KeyValueTable(['Id', 'KeyName', 'Description', 'Hourly', 'Monthly', 'Setup'])
```